### ISSUE TITLE
Reminder that there may be bugs. Best to wait until the official calc is updated.

### ISSUE THREAD

**Issue by Alya-N** [Alya-N/C-Hero-Calc#1]
I've tested a few cases, against doyenne and sparks, but that doesn't mean that the code I wrote is correct. I believe diamond_gizmo is much more experienced with coding and it is a good idea to check if it's been updated here: https://github.com/GizmoMar/C-Hero-Calc

**Comment by robgw84** [comment]
Since there is no Issues tab on that one, I have no choice but to put it here. I dont know if the new battle system they put in is the cause or not, but the calculator cant figure out a correct lineup for my DQ 84
Its: A25, E25, W25, E25, W24
But it gives me only a 5 unit solution that doesnt even work, and it does the calculation in less than a second.

**Comment by Alya-N** [comment]
The most noticeable change in the battle system is the way ricochet works now. It no longer carries over the elemental boost from the first enemy you hit, but counts each ricochet hit as a separate attack (so elemental boost applies to all enemies of the weak element that are hit by ricochet). 
Also, at the time you commented, a bug with Lee occurred when he only reflected damage if he survived the initial attack, that could have been a problem as well (It has been fixed already). 
Those are the two factors I can think of that could lead to a wrong solution, not to mention all the small interactions between skills that may have been changed, yet remain unnoticed. Due to that, it will take a while until a working calculator is released.

**Comment by robgw84** [comment]
Yeah I tried again after the bug was fixed, and it went further, but it came up again. What happens is that in the GUI it says that the solution was accepted but it produces the exact same lineup again, like DQ 90 instead of 91. And if I just run the calc alone, it will give a solution, usually only 5 units, but the solution doesnt work.

I figured it would take time to fix things due to the changes, just letting you know about it.
